contrib/repair-analysis/report.py

```diff
@@ -0,0 +1,316 @@
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+import sys
+import warnings
+
```

Comment:
can you add a docstring up here with instructions of how to generate a report?

eg. "start up firedancer-dev  against testnet with this shredcap config value, pip install requirements, python3 report.py, etc. etc."